### PR TITLE
bump a set of test-related JS libs

### PR DESCRIPTION
**Pull request by mistercrunch**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [x] Build / Development Environment
- [ ] Documentation

### SUMMARY
Mostly upgrading jest as I was trying to tackle issues related to
wrong source-maps reporting in jest. The entrance to the rabbit hole is here: https://github.com/facebook/jest/issues/5446

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -27,7 +27,6 @@ repos:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-docstring-first
-    -   id: check-json
```
Comment (mistercrunch) [inline .pre-commit-config.yaml]: This was giving me trouble with he `.babelrc` changes...

File: superset/assets/.babelrc [hunk 2]
```diff
@@ -17,6 +17,8 @@
  * under the License.
  */
  {
+  "sourceMaps": true,
```
Comment (mistercrunch) [inline superset/assets/.babelrc]: These 2 lines are the actual fix.

File: superset/assets/spec/helpers/shim.js [hunk 3]
```diff
@@ -17,7 +17,8 @@
  * under the License.
  */
 /* eslint no-native-reassign: 0 */
-import '@babel/polyfill';
```
Comment (mistercrunch) [inline superset/assets/spec/helpers/shim.js]: This got deprecated in favor of the new approach described here https://www.npmjs.com/package/@babel/polyfill

File: superset/assets/spec/helpers/shim.js [hunk 4]
```diff
@@ -17,7 +17,8 @@
...
+import 'core-js';
```
Comment (kristw) [inline superset/assets/spec/helpers/shim.js]: Shouldn't this be

```
import "core-js/stable";
```

https://babeljs.io/docs/en/next/babel-polyfill.html

**Comment by mistercrunch** [comment]
@kristw , adjusted to `core-js/stable`

**Review by kristw — APPROVED** [review]

**Pull request merged**